From 81f72208dfa1d4180346ea761e2224735abd6ef3 Mon Sep 17 00:00:00 2001
From: fonz93 <fonz993@hotmail.it>
Date: Tue, 30 Sep 2014 20:01:20 +0200
Subject: [PATCH] remove some frequencies

---
 arch/arm/mach-msm/acpuclock-7627.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-msm/acpuclock-7627.c b/arch/arm/mach-msm/acpuclock-7627.c
index 074c3b2..bcd15fc 100644
--- a/arch/arm/mach-msm/acpuclock-7627.c
+++ b/arch/arm/mach-msm/acpuclock-7627.c
@@ -326,9 +326,9 @@ static struct clkctl_acpu_speed pll0_960_pll1_196_pll2_1200_pll4_1401[] = {
 /* 8625 PLL4 @ 1008MHz with GSM capable modem */
 /* Y300-100 */
 static struct clkctl_acpu_speed pll0_960_pll1_245_pll2_1200_pll4_1008_2p0[] = {
-        { 1, 19200, ACPU_PLL_TCXO, 0, 0, 2400, 3, 0, 30720 },
-        { 1, 65536, ACPU_PLL_1, 1, 3, 8192, 3, 1, 61440 },
-        { 1, 98304, ACPU_PLL_1, 1, 1, 12288, 3, 1, 61440 },
+        { 0, 19200, ACPU_PLL_TCXO, 0, 0, 2400, 3, 0, 30720 },
+        { 0, 65536, ACPU_PLL_1, 1, 3, 8192, 3, 1, 61440 },
+        { 0, 98304, ACPU_PLL_1, 1, 1, 12288, 3, 1, 61440 },
         { 0, 122880, ACPU_PLL_1, 1, 1, 15360, 3, 1, 61440 },
         { 1, 245760, ACPU_PLL_1, 1, 0, 30720, 3, 1, 61440 },
         { 0, 300000, ACPU_PLL_2, 2, 3, 37500, 3, 2, 122880 },
